Dockerfile

```diff
@@ -3,11 +3,23 @@ FROM golang:1.4.2
 MAINTAINER J.C. Jones "jjones@letsencrypt.org"
 MAINTAINER William Budington "bill@eff.org"
 
+# Add node.js and rabbitmq key to apt-key safely
```

Comment:
Looks like you're not setting up the rabbitmq key but using the default debian repo.